.circleci/config.yml

```diff
@@ -13,6 +13,9 @@ jobs:
       - checkout
       - run:
           command: |
+            echo "Making sure docs are updated"
+            make docs
+            [[ -z $(git status -s) ]] || exit 1
```

Comment:
Sending an error message saying that the docs are outdated will be nice

Reply:
Done.